content/components/page-header.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Page header
+description: Page header determines the top-level headings of a UI
+reactID: PageHeader
+---
+
+import {Box, Text} from '@primer/react'
+import ComponentLayout from '~/src/layouts/component-layout'
+export default ComponentLayout
+
+## Usage
+
+Page header is used to determine the arrangement of top-level headings and to determine how elements adopt to different viewports. Page header organizes different elements that apply context and provide navigation for the current UI. 
+
+## Anatomy 
+
+Page header can include a context bar, title bar, description, and contextual navigation (pertaining to the current page/UI). 
+
+## Context bar
+
+By default, the Context bar (including its actions) is only visible on Narrow viewports. This is intended to set context for the user on smaller viewports and provide any actions that fit within the context bar.
+
+### Parent link
+
+The default way to let users navigate up in the hierarchy on Narrow viewports.
+
+**Responsive strategy**: On Narrow viewports, the ParentLink touch target expands beyond its element boundaries, to match with the minTouchTarget design token.
+
+### Context bar actions
+
+Actions that appear in the Context bar specifically.
+
+### Context bar
```

Comment:
The markdown lints fail as there are two identical `Context bar` titles

Reply:
ah thanks for catching!